stubs/olefile/olefile/olefile.pyi

```diff
@@ -222,10 +225,10 @@ class OleFileIO:
     def get_rootentry_name(self) -> bytes: ...
     def getproperties(
         self, filename: str | Sequence[str], convert_time: bool = False, no_conversion: list[int] | None = None
-    ) -> dict[int, list[int | str | bytes | bool | None]]: ...
+    ) -> dict[int, list[int | str | bytes | bool | None] | int | str | bytes | bool | None]: ...
     def _parse_property(
         self, s: bytes, offset: int, property_id: int, property_type: int, convert_time: bool, no_conversion: list[int]
-    ) -> list[int | str | bytes | bool | None] | None: ...
+    ) -> list[int | str | bytes | bool | None] | int | str | bytes | bool | None: ...
```

Comment:
looks like a good candidate for an alias.

Reply:
I've added a commit.